hail/src/main/scala/is/hail/io/fs/FS.scala

```diff
@@ -174,7 +174,7 @@ abstract class FSSeekableInputStream extends InputStream with Seekable {
 
 abstract class FSPositionedOutputStream extends OutputStream with Positioned {
   protected[this] var closed: Boolean = false
-  protected[this] val bb: ByteBuffer = ByteBuffer.allocate(64 * 1024)
+  protected[this] val bb: ByteBuffer = ByteBuffer.allocate(8 * 1024 * 1024)
```

Comment:
This is used for the azure fs as well, so should this optionally be 4Mb?

Reply:
I was going to see if the tests all pass with 8 MB and then make a decision then.

Reply:
Ya looks like it's yelling about the chunk limit https://ci.azure.hail.is/batches/3209370/jobs/121